addons/account/models/account_journal_dashboard.py

```diff
@@ -611,6 +611,9 @@ def _get_open_payments_query(self):
             ('move_id.payment_state', 'in', ('not_paid', 'partial')),
             ('date_maturity', '!=', False),
             ('amount_residual', '<', 0),
+            ('account_id.reconcile', '=', True),
+            ('account_id.account_type', '=', 'liability_payable'),
+            ('display_type', '=', 'payment_term'),
```

Comment:
Why adding the display type ? Isn't just considering the payable account enough ?

(below as well)

Reply:
To be sure to compute the good amount (even if some account configuration are updated by user).

Reply:
@oco-odoo can you give me your best solution ? regards

Reply:
I'd only keep the account_type condition, both here and below